x-pack/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -1101,191 +1107,127 @@ describe('Lens App', () => {
     });
   });
 
-  it('displays errors from the frame in a toast', () => {
```

Comment:
Looks like this test was removed entirely?

Reply:
This was moved up to line `324` so it wasn't so hard to find between 2 nested `describe`s